### ISSUE TITLE
[Bug]: thread 'main' has overflowed its stack

### ISSUE THREAD

**Issue by sitiom** [russellbanks/Komac#977]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Not sure what's happening here:
```console
> komac update c0re100.qBittorrent-Enhanced-Edition -v 5.0.3.10 -u https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/download/release-5.0.3.10/qbittorrent_enhanced_5.0.3.10_x64_setup.exe
Latest version of c0re100.qBittorrent-Enhanced-Edition: 5.0.2.10

thread 'main' has overflowed its stack
```
https://github.com/c0re100/qBittorrent-Enhanced-Edition/actions/runs/12437408221/job/34727199555

**Comment by PckgrTom** [comment]
![image](https://github.com/user-attachments/assets/0821f940-da54-4367-909b-8cdce79f3205)

**Comment by PckgrTom** [comment]
I have been seeing this on a number of submissions, see image above.

**Comment by Dragon1573** [comment]
This issue is still present. See https://github.com/microsoft/winget-pkgs/issues/204531#issuecomment-2566893077.

```text
15:46:49 D:\...\winget-pkgs  [master ≡] 1ms pwsh$ komac --version
komac 2.8.0
```

**Comment by russellbanks** [comment]
> This issue is still present.

I haven't released a komac update with this fix yet. It will be included in komac 2.9.0.
